listView.js

```diff
@@ -13,6 +14,15 @@ const RENDER_AHEAD_DISTANCE = 1000000;
 
 class EnhancedListView extends Component {
   static DataSource = ListView.DataSource;
+  static propTypes = {
+    ...ListView.propTypes,
+    onSectionChanged: PropTypes.func
+  };
+  static defaultProps = {
+    // arbitrarily large distance to pre-render all sections for measurements
+    scrollRenderAheadDistance: 1000000,
```

Comment:
I'm aware, but this is currently needed and the only way to know the measurements, which are used to derive the offset for each section to be used when scrolling to a section